samples/webcam_filter.py

```diff
@@ -44,10 +44,11 @@
             # (https://pysource.com/2018/10/11/how-to-create-a-cartoon-effect-opencv-with-python/)
             # 1) Edges
             gray = cv2.cvtColor(in_frame, cv2.COLOR_BGR2GRAY)
-            gray = cv2.medianBlur(gray, 5)
-            edges = cv2.adaptiveThreshold(gray, 255, cv2.ADAPTIVE_THRESH_MEAN_C, cv2.THRESH_BINARY, 9, 9)
+            gray = cv2.medianBlur(gray, ksize=5)
+            edges = cv2.adaptiveThreshold(gray, maxValue=255, adaptiveMethod=cv2.ADAPTIVE_THRESH_MEAN_C,
```

Comment:
The parameter names come directly from the OpenCV's C++ API. I don't think they can be considered stable or part of the API contract (given that C++ does not have keyword arguments), so I would rather leave them positional.

Reply:
https://docs.opencv.org/3.4/da/d49/tutorial_py_bindings_basics.html: The parameter names correspond to definitions in the c++ header.
E.g. `maxValue`  [header](https://github.com/opencv/opencv/blob/master/modules/imgproc/include/opencv2/imgproc.hpp#L2952).
Thus I guess it is stable enough.